Makefile

```diff
@@ -9,4 +9,21 @@ prepare-release:
 	cd IEKit/; git fetch origin --tags --force; git checkout $$(git tag --sort=committerdate | tail -n 1)
 	cd price-oracle/; git fetch origin --tags --force; git checkout $$(git tag --sort=committerdate | tail -n 1)
 	git submodule status
-	
\ No newline at end of file
+
+
+.SILENT: release-message
+release-message:
+	echo "## v0.0.0" "($(shell date +%Y-%m-%d))"
```

Comment:
Why `echo "## v0.0.0"`?

Reply:
Because we don't have a version tag when we write the release message yet. This is just a helper function, the code has to be copy-pasted in the HISTORY.md and change the version number accordingly. Then, when the commit is merged, the tag can be created.